cmd/gapit/graph_visualization.go

```diff
@@ -0,0 +1,72 @@
+// Copyright (C) 2018 Google Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	"github.com/google/gapid/core/app"
+	"github.com/google/gapid/core/log"
+	"path/filepath"
+	"io/ioutil"
+	"context"
+	"flag"
+)
+
+type graph_visualizationVerb struct{ GraphVisualizationFlags }
+
+func init() {
+	verb := &graph_visualizationVerb{}
+	app.AddVerb(&app.Verb{
+		Name:      "graph_visualization",
+		ShortHelp: "Get and Write graph-visualization file from capture",
+		Action:    verb,
+	})
+}
+
+func (verb *graph_visualizationVerb) Run(ctx context.Context, flags flag.FlagSet) error {
+	if flags.NArg() != 1 {
+		app.Usage(ctx, "Exactly one gfx capture file expected, got %d", flags.NArg())
+		return nil
+	}
+
+	capturePath, err := filepath.Abs(flags.Arg(0))
```

Comment:
I recommend you stick with the pattern we are using elsewhere for GAPIS.
https://github.com/google/gapid/blob/master/cmd/gapit/screenshot.go#L62

It lets you interact with an already-open gapis & trace.